css/style.css

```diff
@@ -0,0 +1,615 @@
+*,
+*::after,
+*::before {
+  box-sizing:border-box }
+
+body {
+  max-width: 1350px;
+  margin: 0 auto!important;
+  background-color: rgb(253, 249, 249)!important;
+}
+
+html, body {
+    overflow-x: hidden;
+}
+.container-fluid {
+    position: relative;
+    background-color: white
+}
+
+.heading-none{
+    display: none
+}
+
+.nav-bar {
+	position: relative;
+	margin: auto;
+	background-color: #f72210;
+	color: #fff;
+	padding: 50px 0px 10px 0px;
+	max-width: 1310px;
+}
+
+
+.nav-second {
+	position: relative;
+	max-width: 1310px;
+	margin: auto;
+}
+
+.logoimg {
+	position: absolute;
+	width: 396px;
+	height: 60px;
+	transform: translate(-50%, 0);
+	top: 15px;
+	left: 50%;
+}
+.logoimg > img {
+	width: 100%;
+}
+
+.datenav {
+	position: relative;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.